Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.h

```diff
@@ -325,6 +334,7 @@ class PDFPluginBase : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<PDF
     uint64_t m_streamedBytes { 0 };
 
     RetainPtr<PDFDocument> m_pdfDocument;
+    RetainPtr<WKAccessibilityPDFDocumentObject> m_accessibilityObject;
```

Comment:
Maybe call this `m_accessibilityDocumentObject` since it represents the document.